models/players.js

```diff
@@ -19,12 +19,13 @@ var shortId = require('shortid');
 
 var returnPlayers = function (players, rb_team_id, res, league){
   if (res.quiz_page != undefined && res.quiz_page){
-    var roster = [];
-     starters = formatEvenOdds(players.starters, true),
-     bench = formatEvenOdds(players.bench);
+    var roster = {};
+     starters = formatEvenOdds(players.players.starters, true),
+     bench = formatEvenOdds(players.players.bench);
+     roster.bench = bench;
+     roster.starters = starters;
```

Comment:
Can/should this condense to?

roster.starters = formatEvenOdds(players.players.starters, true)
roster.bench = format...